spec/index.bs

```diff
@@ -3850,6 +3850,7 @@ A {{GPUBindGroup}} object has the following internal slots:
                                             - |layoutBinding|.{{GPUBindGroupLayoutEntry/storageTexture}}.{{GPUStorageTextureBindingLayout/format}}
                                                 is equal to |resource|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}.
                                             - |texture|'s {{GPUTextureDescriptor/usage}} includes {{GPUTextureUsage/STORAGE_BINDING}}.
+                                            - |resource|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/mipLevelCount}} must be 1.
```

Comment:
This seems like a strange place to handle this. If we want to disallow storage textures with mipmaps, we should do it at `createTexture` time based on the presence of "STORAGE_BINDING" flag

Reply:
IIUC storage textures with mipmaps are supposed to be allowed. It's just that texture views with multiple layers can't be used as storage bindings. (They can still be used for other things like sampled bindings or copies.)